src/proxyprotocol/haproxy_test.go

```diff
@@ -0,0 +1,88 @@
+package proxyprotocol_test
```

Comment:
This file should probs be renamed.

Reply:
I did `grep -ir haproxy src` and found I'd fixed all the references.

But I totally missed the file name.

Fixed.